library/modules/bigip_profile_http.py

```diff
@@ -0,0 +1,670 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2018, F5 Networks Inc.
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: bigip_profile_http
+short_description: Manage HTTP profiles on a BIG-IP
+description:
+  - Manage HTTP profiles on a BIG-IP.
+version_added: 2.7
+options:
+  name:
+    description:
+      - Specifies the name of the profile.
+    required: True
+  parent:
+    description:
+      - Specifies the profile from which this profile inherits settings.
+      - When creating a new profile, if this parameter is not specified, the default
+        is the system-supplied C(http) profile.
+  description:
+    description:
+      - Description of the profile.
+  proxy_type:
+    description:
+      - Specifies the proxy mode for the profile, 
+      - When creating a new profile, if this parameter is not specified, the
+        default is provided by the parent profile.
+    choices:
+        - reverse
+        - transparent
+        - explicit
+  dns_resolver:
+    description:
+      - Specifies the name of a configured DNS resolver, this option is mandatory when C(proxy_type)
+        is set to C(explicit).
+      - Format of the name can be either be prepended by partition: C(/Common/foo) ,or specified
```

Comment:
comma has a space before it and is missing a space after it. Additionally, do not use `:` in the description because it will cause the docs generator to fail. You can switch the : to just wrapping the trailing string in ().

For example, (C(/Common/foo))